app/views/shared/menu/_cart_sidebar.html.haml

```diff
@@ -0,0 +1,41 @@
+.expanding-sidebar.cart-sidebar{ng: {controller: 'CartCtrl', show: 'showCartSidebar', class: "{'shown': showCartSidebar, 'hidden': !showCartSidebar}"}}
+  .background{ng: {click: 'toggleCartSidebar()'}}
+  .sidebar
+    .cart-header
+      %span.title{"ng-show" => "Cart.line_items.length > 0"}
+        = t('.items_in_cart', num: "{{ Cart.total_item_count() }}")
+      %a.close{ng: {click: 'toggleCartSidebar()'}}
+        = t('.close')
+        %i.ofn-i_009-close
+
+    .cart-content
+      %table
+        %tr.product-cart{"ng-repeat" => "line_item in Cart.line_items", "id" => "cart-variant-{{ line_item.variant.id }}"}
+          %td.image
+            %img{'ng-src' => '{{ line_item.variant.thumb_url }}'}
+          %td
+            %span {{ line_item.variant.extended_name }}
+            %br
```

Comment:
why don't we use a block element instead of a `span` so there's no need for a `br`? the need for it clearly shows the `span` doesn't fit the needs.